### PR TITLE
[Forge Cli] Small improvements for running a local swarm.

### PR DESCRIPTION
**Pull request by JoshLind**
### Description

This PR makes some small improvements to the `forge-cli` tool to make it possible to run a local swarm (with a specified number of validator nodes) and interact with it. The PR also adds a small README for tool usage (we'll likely want to expand this as time goes on). With this PR, we now have a starting point for local swarm deployments that we can begin to play with, test, benchmark, attack etc.

### Test Plan
Ran the tool manually:

```
Starting Swarm with supported versions: ["b28cafd3c02ae709886f147d93efc51ec645f4da"]
Preparing a new swarm
Building a new swarm
Building genesis with 3 validators. Directory of output: "/private/var/folders/dx/c0l2rrkn0656gfx6v5_dy_p80000gn/T/.tmpDLHaaE"
2022-08-03T21:40:10.918416Z [main] INFO storage/schemadb/src/lib.rs:291 Opened RocksDB. {"rocksdb_name":"ledger_db"}
2022-08-03T21:40:10.930857Z [main] INFO storage/schemadb/src/lib.rs:291 Opened RocksDB. {"rocksdb_name":"state_merkle_db"}
2022-08-03T21:40:10.931849Z [main] INFO storage/aptosdb/src/lib.rs:388 Opened AptosDB (LedgerDB + StateMerkleDB). {"ledger_db_path":"/var/folders/dx/c0l2rrkn0656gfx6v5_dy_p80000gn/T/ea6a2d37af46e456eb6335388beac601/ledger_db","state_merkle_db_path":"/var/folders/dx/c0l2rrkn0656gfx6v5_dy_p80000gn/T/ea6a2d37af46e456eb6335388beac601/state_merkle_db","time_ms":"156"}
2022-08-03T21:40:10.932760Z [main] INFO aptos-move/aptos-vm/src/adapter_common.rs:170 Executing block, transaction count: 1 {"name":"miscellaneous","txn_id":0}
2022-08-03T21:40:10.937058Z [main] INFO aptos-move/aptos-vm/src/adapter_common.rs:224 Reconfiguration occurred: restart required {"name":"miscellaneous","txn_id":0}
2022-08-03T21:40:10.953713Z [main] INFO execution/executor/src/db_bootstrapper.rs:190 Genesis calculated: ledger_info_with_sigs Some(V0(LedgerInfoWithV0 { ledger_info: LedgerInfo { commit_info: BlockInfo { epoch: 0, round: 0, id: HashValue(0000000000000000000000000000000000000000000000000000000000000000), executed_state_id: HashValue(7a41614c791112a1c5f4b6caafa8f19aa35cb99f8c26b8e7970d1cc6b3084be0), version: 0, timestamp_usecs: 0, next_epoch_state: Some(EpochState [epoch: 1, validator: ValidatorSet: [35300861a92125573cfe5d38965bcef3bade3b22e569285d44818bc3e513e043: 1, 43cb9675dba04e41902ad97dfb70bb1d476a99c7c1ab357bb451ecade0c9fc89: 1, e752ad2f84520e475a5fbff6ed863073b713880e702e8ff6c10cac446f4c12f3: 1, ]]) }, consensus_data_hash: HashValue(0000000000000000000000000000000000000000000000000000000000000000) }, signatures: {} })), waypoint Waypoint { version: 0, value: HashValue(e458207acced3be3c699e621ec7f9f0548a18f2cd01c9bcaa6b02ae3a2a27115) }
Started node "0" (PID: 3505) with command: ".../aptos-core/target/debug/aptos-node" "-f" "/private/var/folders/dx/c0l2rrkn0656gfx6v5_dy_p80000gn/T/.tmpDLHaaE/0/node.yaml"
Started node "1" (PID: 3506) with command: ".../aptos-core/target/debug/aptos-node" "-f" "/private/var/folders/dx/c0l2rrkn0656gfx6v5_dy_p80000gn/T/.tmpDLHaaE/1/node.yaml"
Started node "2" (PID: 3507) with command: ".../aptos-core/target/debug/aptos-node" "-f" "/private/var/folders/dx/c0l2rrkn0656gfx6v5_dy_p80000gn/T/.tmpDLHaaE/2/node.yaml"
Wait for startup attempt: 0 of 10
health check failure: error sending request for url (http://localhost:57536/forge_metrics): error trying to connect: tcp connect error: Connection refused (os error 61)
Wait for startup attempt: 1 of 10
health check failure: error sending request for url (http://localhost:57536/forge_metrics): error trying to connect: tcp connect error: Connection refused (os error 61)
Wait for startup attempt: 2 of 10
Swarm alive.
Swarm launched successfully.
The network has been deployed. Hit Ctrl+C to kill this, otherwise it will run forever.
```

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/2503)

### CODE REVIEW

File: testsuite/forge-cli/src/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+id: forge cli
+title: Forge CLI
+custom_edit_url: https://github.com/aptos-labs/aptos-core/edit/main/testsuite/forge-cli/README.md
+---
+
+# Forge CLI
```
Comment (rustielin) [inline testsuite/forge-cli/src/README.md:7]: awesome! I'm planning on moving some of our internal docs to open source as well https://www.notion.so/aptoslabs/Forge-Runbook-80906ba523e2467e9efab9771caf9a91. It's mostly the k8s stuff, but we can combine into this README.

**Review by rustielin — APPROVED** [review]
Amazing! LGTM

File: testsuite/forge-cli/src/README.md (hunk 1 above)
Comment (JoshLind) [reply testsuite/forge-cli/src/README.md:7]: Awesome! That would be great 😄

**Comment by rustielin** [comment]
Re-running the forge job...

**Pull request merged**